src/imitation/scripts/parallel.py

```diff
@@ -104,6 +137,18 @@ def _ray_tune_sacred_wrapper(
     """
 
     def inner(config: dict, reporter) -> dict:
+        """Trainable function with the correct signature for `ray.tune`.
+
+        Args:
+            config: Keyword arguments for `ex.run()`, where `ex` is the
+                `sacred.Experiment` instance associated with `sacred_ex_name`.
+        """
+        # Set Sacred capture mode to "sys" because default "fd" option leads to error.
```

Comment:
I'd make this a TODO or SOMEDAY since hopefully one day the Sacred issue is resolved and we can remove this workaround